ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml

```diff
@@ -86,7 +86,10 @@ tests:
   cron: '@yearly'
   steps:
     cluster_profile: ibmcloud-qe
-    workflow: cucushift-installer-rehearse-ibmcloud-ipi-minimal-permission
+    env:
+      APPLY_ALL_SUBNETS: "yes"
+      NUMBER_SUBNETS_PAIR_PER_ZONE: "2"
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8
```

Comment:
Why here using a rhel worker install workflow, but not a simpler install workflow ?